README.rst

```diff
@@ -108,8 +108,8 @@ The repository consists mostly of externally hosted subrepositories:
 .. _libtiff: https://gitlab.com/libtiff/libtiff
 .. |libtifflic| replace:: BSD-2 license
 .. _libtifflic: https://gitlab.com/libtiff/libtiff/-/blob/master/README.md
-.. |libtiffver| replace:: 4.4.0 (+ Build System Patch)
-.. _libtiffver: https://gitlab.com/libtiff/libtiff/-/tree/v4.4.0
+.. |libtiffver| replace:: 4.5.0 (+ Build System Patch)
```

Comment:
I think it has been done already in https://github.com/NVIDIA/DALI_deps/pull/74/files ?

Reply:
True, don't know what happened. Will update